test/lit/passes/optimize-added-constants-memory64.wast

```diff
@@ -0,0 +1,54 @@
+;; RUN: wasm-opt %s -all --roundtrip --optimize-added-constants --low-memory-unused -S -o - | filecheck %s
```

Comment:
```suggestion
;; NOTE: Assertions have been generated by update_lit_checks.py and should not be edited.

;; RUN: wasm-opt %s -all --roundtrip --optimize-added-constants --low-memory-unused -S -o - | filecheck %s
```
Adding that first magical line will make `./scripts/auto_update_tests.py lit` auto-generate the CHECK lines. It will also place each CHECK before the relevant function etc. in a more readable way.

Reply:
address the comment.